STREAMS-2322 Update kafka version and update installation documentation accordingly (#129)
Axway/streams-open-docs: content/en/docs/relnotes/20211130_streams_relnotes.md

The following third-party library has been upgraded:

* MariaDB : `10.4.21`
* Apache Kafka : `2.8.1`

### Kafka upgrade

> Due to security vulnerabilities, we had to upgrade Apache Kafka to 2.8.1. This update introduces breaking changes in the configuration of embedded kafka.

Reviewer: ```suggestion
Due to security vulnerabilities, Apache Kafka has been upgraded to version 2.8.1. This upgrade introduces breaking changes in the configuration of embedded kafka.
```

> SASL configuration are now under `embeddedKafka.auth.sasl`. For more information, see [Helm parameters](/docs/install/helm-parameters/#mariaDB-parameters).

Reviewer: ```suggestion
SASL configuration are now under `embeddedKafka.auth.sasl`. For more information, see [Helm parameters](/docs/install/helm-parameters/#kafka-parameters).
```
